Makefile

```diff
@@ -93,6 +93,15 @@ release-publish-local:
 	make release-local
 	make publish-local
 
+clone-swiftline:
+	@echo "Removing old Swiftline"
+	@echo ""
+	rm -rf SwiftLineTemp
+
+	@echo "Clone new Swiftline from oarrabi/linux! branch"
+	git clone -b oarrabi/linux! https://github.com/oarrabi/Swiftline.git SwiftLineTemp
```

Comment:
The branch in Swiftline is about making it work on linux https://github.com/oarrabi/Swiftline/pull/30

I think we should keep the name until we merge it